python/py-nbconvert/Portfile

```diff
@@ -60,5 +60,16 @@ if {${name} ne ${subport}} {
                             port:py${python.version}-nbclient
     }
 
+    notes "
+To convert markdown to formats other than HTML, nbconvert uses pandoc. \
+Install it by running
+
+sudo port install pandoc
+
+To convert notebooks to PDF, nbconvert makes use of LaTeX and the XeTeX \
+as the rendering engine. Install it by running
+
+sudo port install texlive-xetex
+"
```

Comment:
you shouldn't have a <space> before the `\` in notes; other than that looks good to me.

Reply:
a whitespace. you mean?

Reply:
Ok, updated (somehow GitHub markdown swallowed your `<space>` :D